plugins/home/src/homePageComponents/WelcomeTitle/WelcomeTitle.test.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 The Backstage Authors
+ * Copyright 2023 The Backstage Authors
```

Comment:
You can leave this, I think the general pattern is that it represents when the code was first written. But let's let the maintainers confirm this one

Reply:
fixed